server/scripts/2020TaxSeasonDataScript.php

```diff
@@ -0,0 +1,409 @@
+<?php
+// This script will only work for Matthew Meacham
+
+$root = realpath($_SERVER['DOCUMENT_ROOT']);
+require_once "$root/server/user.class.php";
+$USER = new User();
+if (!$USER->isLoggedIn() || $USER->getUserId() !== '1') {
+	header("Location: /unauthorized");
+	die();
+}
+
+// TODO: Need to update the `$dataAlreadyInserted` values before this query will run successfully
+
+
+insert2020Data();
+
+function insert2020Data() {	
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The data has already been inserted');
+	}
+
+	// insertAndersonLibraryData();
+	// insertCenterForPeopleInNeedData();
+	// insertLorenEiseleyLibraryData();
+	// insertBennettMartinLibraryData();
+    // insertFStreetCommunityCenterData();
+    // insertSoutheastCommunityCollegeData();
+	die('SUCCESS');
+}
+
+// Need to create the new sites
+// Create Shifts for each site
+// Create appointment times within those shifts
+// Create role limits for that site/shift/role
+
+function insertAndersonLibraryData() {
+	GLOBAL $DB_CONN;
+
+	$siteCoordinatorRoleId = 1;
+	$greeterRoleId = 2;
+	$preparerRoleId = 3;
+    $reviewerRoleId = 4;
+    
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The AL data has already been inserted');
+	}
+
+	$siteId = 2; // Manually obtained from PROD DB
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		// Wednesdays
+		$dates = getWeeklyDatesFromRange('2020-01-29', '2020-04-08');
+		foreach ($dates as $date) {
+			$shiftId = insertShift("$date 17:00:00", "$date 20:00:00", $siteId);
+
+            // Appointment numbers are from per Katie's parameter sheet
+			$firstAppointmentTimeId = insertAppointmentTime("$date 17:00:00", 9, 9, 100, 60, $siteId);
+            $secondAppointmentTimeId = insertAppointmentTime("$date 18:00:00", 9, 9, 100, 60, $siteId);
+            $thirdAppointmentTimeId = insertAppointmentTime("$date 19:00:00", 5, 5, 100, 60, $siteId);
+        }
+        
+        // Thursdays
+        $thursdayDates = getWeeklyDatesFromRange('2020-01-30', '2020-04-09');
+        foreach ($dates as $date) {

```

Comment:
This should be `$thursdayDates as $date` -- currently, it's using the `$date` variable from above which would be your Wednesday dates... That brings up a good point, maybe rename `$dates` anywhere to be `$[day]Dates` -- that's read clearer, and hopefully avoid small errors like this

Reply:
Done.